logfire/_internal/integrations/llm_providers/anthropic.py

```diff
@@ -83,9 +83,14 @@ def on_response(response: ResponseT, span: LogfireSpan) -> ResponseT:
     return response
 
 
-def is_async_client(client: type[anthropic.Anthropic] | type[anthropic.AsyncAnthropic]):
-    """Returns whether or not the `client` class is async."""
-    if issubclass(client, anthropic.Anthropic):
-        return False
-    assert issubclass(client, anthropic.AsyncAnthropic), f'Expected Anthropic or AsyncAnthropic type, got: {client}'
-    return True
+def is_async_client(client_type: Any) -> bool:
+    """Returns whether the `client` class is async."""
+    if isinstance(client_type, type):
```

Comment:
`is_async_client` is already only called with a class as the argument.

Reply:
Removed in latest